### PR TITLE
Cleaning summaries

### PR DESCRIPTION
**Pull request by tomkocmi**

### CODE REVIEW

**Comment by tomkocmi** [comment]
Odstranil jsem summaries, ktere jsou zbytecne duplicitni, runner automaticky bere train_loss a vypisuje ho do summaries, proto byly zbytecne. Navic az se rozhodne co s #440 tak to zde chci dokoncit.

**Comment by jlibovicky** [comment]
Počkej, počkej, to není stejné. Runner vypisuje train_loss na trénovacích datech, to je pravda, ale je to run-time hodnota (ta se hodí k tomu, aby se porovnala na trénovacích datech), ne přímo ta hodnota, které se používá při trénování (tj. s dropoutem a případně ovlivněnou nějakými dalšími triky typu různých renormalizací nebo residual connections). Tuhle hodnotu potřebujeme vidět, abysme viděli, jestli ji optimizer skutečně snižuje. (Byl tam taky bug v batch normalizaci v CNN, který dělal, že ta train-time loss klesala, ale run-time loss rostla i na trénovaích datech.)

Možná stojí za rozmyšlení, jestli se nemají přestěhovat do traineru, ale pryč určitě nemůžou.

**Comment by jindrahelcl** [comment]
v tom pripade by se to melo vratit do decoderu, ale nevim ted jak to presne je

**Comment by tomkocmi** [comment]
No ale tyhle summaries z nejakeho duvodu se nikde neukladaji.
